### ISSUE TITLE
The reset function does not work ?

### ISSUE THREAD

**Issue by Bricebr** [TheInformationLab/Tableau-Extension-ExportAll#18]
Under TDesktop, I used the measure names and values to display data in a worksheet, then add it on a dashboard, publish to the server to add this awesome extension, then allowed it when popuped and clicked to "Modify" under the TS before configured it, to be able to save the settings inside the workbook, on TS. 
I wish I'd read the trouble with the measure N&V displayed in one unique column before.
Now, while the export button works fine, I need to change the measure N&V but...
no "rescan function" to update changes on the worksheet, sad it is,
and the Reset button just erase the "Select Sheets" and "Select Columns" contents, nothing is displayed anymore wherease the "Configure" section is properly reset to its original values.
well.. "Same player, shoot again" I guess. That means the configure part must be done as the very last thing on a approved workbook, unless Sisyphe.

**Comment by andre347** [comment]
No clear issue description.

**Comment by Bricebr** [comment]
Ok, let me try again.
on my dashboard, I had 3 sheets with respectively 10, 8 and 3 columns to configure. I did with the "configure" menu of the extension, saved and closed it, and saved & closed the dashboard. Now I want to change the initial settings, so I re-open the dashboard in edit mode, do a configure from the panel menu, and click on the reset button : everything is cleared => no sheets anymore to select, so no column either, only the name and look&feel of the "Export All" button is displayed.
